spec/models/user_spec.rb

```diff
@@ -269,4 +269,37 @@
       end
     end
   end
+
+  describe "delete" do
+    let(:user) { FactoryBot.create(:user) }
+    let!(:owned_lettings_log) do
+      FactoryBot.create(
+        :lettings_log,
+        :completed,
+        owning_organisation: user.organisation,
+        managing_organisation: user.organisation,
+        created_by: user,
+      )
+    end
+
+    context "when the user is deleted" do
+      it "owned lettings logs are not deleted as a result" do
+        expect(described_class.count).to eq(1)
+        expect(LettingsLog.count).to eq(1)
+        user.destroy!
+        expect(described_class.count).to eq(0)
+        expect(LettingsLog.count).to eq(1)
+      end
+
+      it "owned lettings logs are not deleted as a result" do
```

Comment:
This looks like it's testing that they are correctly deleted when the organisation is deleted? In that case might this test be better in the Organisation spec? (Although I think one maybe already exists)